packages/edit-site/src/components/global-styles/shadows-panel.js

```diff
@@ -136,11 +193,9 @@ function ShadowItem( { shadow, category } ) {
 		<NavigationButtonAsItem
 			path={ `/shadows/edit/${ category }/${ shadow.slug }` }
 		>
-			<HStack>
+			<HStack justify="space-between">
```

Comment:
I don't think `justify="space-between"` is required, as it's the default alignment for this component.